ansible/roles/openslide/tasks/main.yml

```diff
@@ -88,14 +90,19 @@
 
 # libtiff
 - name: Download libtiff
-  get_url:
-    url: http://download.osgeo.org/libtiff/tiff-4.0.6.tar.gz
-    dest: "{{ root_dir }}"
+
+  command: >-
+    wget --retry-connrefused --waitretry=1 --read-timeout=20 http://download.osgeo.org/libtiff/tiff-4.0.6.tar.gz
```

Comment:
I'd also recommend explicitly specifying the output file name (`-O tiff-4.0.6.tar.gz`), since if this is run as a local provisioning step and the download is partial the first time, the second time a file called `tiff-4.0.6.tar.gz.1` will be created, and the next step will never pass.